bioimageio/spec/commands.py

```diff
@@ -57,9 +57,9 @@ def validate(
             source_name = str(rdf_source)
         except Exception as e:
             source_name = str(e)
-
-    if not isinstance(rdf_source, dict):
-        error = f"expected loaded resource to be a dictionary, but got type {type(rdf_source)}: {str(rdf_source)}"
+    else:
```

Comment:
I think `elif not ....` would be more readable here.

Reply:
unfortunately the `else` of a `try/except` block can not be an `elif`.

Reply:
Maybe I am missing something, but as far as I can see this `else` is not linked to any `try/except` block at all, but only relates ti the `if not isinstance ...` above.
Afaik there is no `try/except/if/else` construction ...

Reply:
there are two try/except blocks, the outer one has the else: https://github.com/bioimage-io/spec-bioimage-io/blob/29446bd9b8015513705445c11a507f79f6c2dbfd/bioimageio/spec/commands.py#L51-L62